src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -31,10 +31,22 @@ public override ExportResult Export(in Batch<Activity> batch)
                 this.WriteLine($"Activity.ParentSpanId:       {activity.ParentSpanId}");
             }
 
-            this.WriteLine($"Activity.ActivitySourceName: {activity.Source.Name}");
+            this.WriteLine($"Activity.ActivitySource.Name: {activity.Source.Name}");
             if (!string.IsNullOrEmpty(activity.Source.Version))
             {
-                this.WriteLine($"Activity.ActivitySourceVersion: {activity.Source.Version}");
+                this.WriteLine($"Activity.ActivitySource.Version: {activity.Source.Version}");
+            }
+
+            if (activity.Source.Tags?.Any() == true)
+            {
+                this.WriteLine("Activity.ActivitySource.Tags:");
```

Comment:
nit: Just like you mentioned in changelog, maybe lets mention that this is InstrumentationScope, so people will be able to easily related (when comparing OTel Collector's console output)